src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped work!");
```

Comment:
Similarly, the message in `stopWork` should be more informative. For example, 'Truck has stopped working.' would be clearer.